fedoracommunity/connectors/bodhiconnector.py

```diff
@@ -525,13 +525,28 @@ def query_active_releases(self, filters=None, **params):
             filters = dict()
         filters = self._query_updates_filter.filter(filters, conn=self)
         package = filters.get('package')
-        pkgdb = get_connector('pkgdb')
         koji = get_connector('koji')._koji_client
         koji.multicall = True
 
-        for release in pkgdb.get_fedora_releases():
-            tag = release[0]
-            name = release[1]
+        releases_obj = self.call('releases', {})
+        total_pages = releases_obj['pages']
+        releases_all = releases_obj['releases']
+        if(total_pages > 1):
+            for i in range(2,total_pages+1):
+                temp = self.call('releases?page='+str(i), {})['releases']
+                releases_all.extend(temp)
+        releases_all.append({'dist_tag': 'rawhide',
+                             'long_name':'Rawhide',
+                             'stable_tag': 'rawhide',
+                             'testing_tag': 'no_testing_tag_found',
+                             'state': 'current'})
+        releases_all = sorted(releases_all, key=lambda k: k['dist_tag'], reverse=True)
```

Comment:
I was previously sorting by 'release' and hence we had a problem displaying rawhide. However, now I am sorting it by 'dist_tag' and hence that problem has disappeared.